src/sentry/utils/performance_issues/detectors/consecutive_db_detector.py

```diff
@@ -194,7 +193,7 @@ def _set_independent_spans(self, spans: list[Span]) -> None:
         To start, we are just checking for a span in a list of consecutive span without a WHERE clause
         """
         independent_spans = []
-        for span in spans[1:]:
+        for span in spans:
```

Comment:
From what I understand, if the spans are independent it shouldn't matter if we check all instead of chopping out the first.